public/js/reducers/ui.js

```diff
@@ -29,7 +29,7 @@ const State = makeRecord(({
 
 function update(state = State(), action: Action) : Record<SidebarsState> {
   switch (action.type) {
-    case C.COLLAPSE_SIDEBAR: {
+    case C.TOGGLE_SIDEBAR: {
```

Comment:
@wldcordeiro no worries! We used to import functions directly like `let { Map } = require("immutable")` but that overrides the native map, so we made it a shorthand, but most things don't use that format.